app/src/main/java/org/mozilla/fenix/search/SearchFragment.kt

```diff
@@ -91,8 +91,10 @@ class SearchFragment : Fragment(), BackHandler {
                     showShortcutEnginePicker = displayShortcutEnginePicker,
                     searchEngineSource = currentSearchEngine,
                     defaultEngineSource = currentSearchEngine,
-                    showSuggestions = Settings.getInstance(requireContext()).showSearchSuggestions,
-                    showVisitedSitesBookmarks = Settings.getInstance(requireContext()).shouldShowVisitedSitesBookmarks,
+                    showSuggestions = Settings.getInstance(requireContext()).shouldShowSearchSuggestions,
```

Comment:
Looks good! Just a minor nit, I think we should take this opportunity to rename `showSuggestions` to `showSearchSuggestions` just to make it a bit clearer.

Reply:
Great point--in fact the kdoc comment above it even specifies it's a `search suggestion` not just a `suggestion`. Updated 😄